ui-admin/src/study/notifications/NotificationContent.tsx

```diff
@@ -54,41 +52,45 @@ export default function NotificationContent({ studyEnvContext, portalContext }:
 
   return <div className="container-fluid px-4 py-2">
     { renderPageHeader('Participant email configuration') }
-    {isLoading && <LoadingSpinner/>}
-    {!isLoading && <div className="row">
-      <div className="col-md-3 mh-100 bg-white border-end">
-        <ul className="list-unstyled p-2">
-          { CONFIG_GROUPS.map(group => <li key={group.type}>
-            <h6 className="pt-2">{group.title}</h6>
-            <ul className="list-unstyled p-2">
-              { configList
-                .filter(config => config.notificationType === group.type)
-                .map(config => <li key={config.id} className="py-1">
-                  <div className="d-flex">
-                    <NavLink to={`configs/${config.id}`} style={navStyleFunc}>
-                      <NotificationConfigTypeDisplay config={config}/>
-                      <span className="text-muted fst-italic"> ({deliveryTypeDisplayMap[config.deliveryType]})</span>
-                    </NavLink>
-                  </div>
-                </li>
-                ) }
-            </ul>
+    <div className="d-flex">
+      {isLoading && <LoadingSpinner/>}
+      {!isLoading && <div style={navDivStyle}>
+        <ul className="list-unstyled">
+          { CONFIG_GROUPS.map(group => <li style={navListItemStyle}>
+            <CollapsableMenu header={group.title} headerClass="text-black" content={
+              <ul className="list-unstyled p-2">
+                { configList
+                  .filter(config => config.notificationType === group.type)
+                  .map(config => <li key={config.id} className="mb-2">
+                    <div className="d-flex">
+                      <NavLink to={`configs/${config.id}`} style={navLinkStyleFunc}>
+                        <NotificationConfigTypeDisplay config={config}/>
+                        <span
+                          className="text-muted fst-italic"> ({deliveryTypeDisplayMap[config.deliveryType]})</span>
+                      </NavLink>
+                    </div>
+                  </li>
+                  ) }
+              </ul>}
+            />
           </li>)}
+          <li style={navListItemStyle} className="ps-3">
+            <button className="btn btn-secondary" onClick={() => setShowCreateModal(true)}>
+              <FontAwesomeIcon icon={faPlus}/> Add
+            </button>
+          </li>
```

Comment:
```suggestion
          { currentEnv.environmentName == 'sandbox' && <li style={navListItemStyle} className="ps-3">
            <button className="btn btn-secondary" onClick={() => setShowCreateModal(true)}>
              <FontAwesomeIcon icon={faPlus}/> Add
            </button>
          </li> }
```

This is an action that our users can only perform in the `sandbox` study environment, so we should hide the button in those other cases